CONTRIBUTING.md

```diff
@@ -22,6 +23,14 @@
 -   _topic_ branches are:
     -   based on `development` and will be squash-merged into `development`.
 
+### Merging `development` into `master`
+
+We're following Gitflow: _topic_ branches (feature or bug-fix) -> `development` -> `master`.
+
+-   _topic_ branches: based on development and implement changes for new features or bug fix.
+-   `development`: main development branch where all topic branches are merged for ongoing development.
+-   `master`: production branch of what has been released. Tags will also be created for release versions.
```

Comment:
Sorry, maybe I wasn't clear. Didn't expect you to copy and paste what I wrote, I expected instructions of how we merge changes from the development branch into the master branch when a development cycle finishes.

Reply:
Done